### ISSUE TITLE
[Feature Request]Dehumidifier supporting

### ISSUE THREAD

**Issue by huhsian** [ollo69/ha-smartthinq-sensors#148]
Hello?

Is any chance to add support for Dehumidifier? I have 2 humidifiers which are connected to ver 2 server. There are integrations in Samsung smartthings (LG connector : https://github.com/fison67/LG-Connector), but no support in HA.

Thank you,
Best Regards

**Comment by KTibow** [comment]
You'll need to do some capturing of what the app sends to the server and vice versa.

**Comment by GeneFung** [comment]
I have a LG humidifiers too and am looking forward to it.
Since I'm new to this, if you show me how to capture the package, I might help.

**Comment by KTibow** [comment]
I don't know myself; @ollo69 might know how though.

**Comment by KTibow** [comment]
Also, this may be a duplicate of #102.

**Comment by DrewL4** [comment]
Has support been added yet?

**Comment by ollo69** [comment]
No, still not found time to do this. Is in my to do list, I will report here when ready...

**Comment by ollo69** [comment]
Dehumidifier support is now available with last release. I close this FR, please refer to FR #318 for feed-back
